README.md

```diff
@@ -164,27 +164,13 @@ For OS/distribution specific details see see [wiki].
 
 ## 3. Configuration
 
-For a step-by-step initial configuration guide, see [getting-started]
-
 User config files are located at `${XDG_CONFIG_HOME:-$HOME/.config/labwc/}`
-with the following four files being used:
-
-| file               | man page
-| ------------------ | --------
-| [rc.xml]           | [labwc-config(5)], [labwc-actions(5)]
-| [menu.xml]         | [labwc-menu(5)]
-| [autostart]        | [labwc(1)]
-| [environment]      | [labwc-config(5)]
-| [themerc-override] | [labwc-theme(5)]
-
-The example [rc.xml] has been kept simple. For all options and default values,
-see [rc.xml.all]
-
-Configuration and theme files are reloaded on receiving SIGHUP
-(e.g. `killall -s SIGHUP labwc`)
+with the following five files being used: [rc.xml], [menu.xml], [autostart],
+[environment] and [themerc-override].
 
-For keyboard settings, see [environment] and [xkeyboard-config(7)]
+Run `labwc --reconfigure` to reload configuration and theme.
 
+For a step-by-step initial configuration guide, see [getting-started]
```

Comment:
The dot at the end of the line is missing.

Reply:
Thanks